packages/cli/commands/project/download.js

```diff
@@ -67,7 +67,7 @@ exports.handler = async options => {
         })
       );
       let { name: promptedProjectName } = await downloadProjectPrompt(options);
-      let projectName = promptedProjectName || project;
+      projectName = promptedProjectName || project;
```

Comment:
This is unrelated to the account not being honored, but this was redeclaring a variable defined in the parent scope [here](https://github.com/HubSpot/hubspot-cli/pull/1080/files#diff-a746cff94477f3abe62a319178fa6394252aed5473144e39412aa31af3f87d7aR46)